databus-api/src/test/java/com/bazaarvoice/emodb/databus/api/SubscriptionJsonTest.java

```diff
@@ -48,21 +47,6 @@ public void testSubscriptionJson() throws Exception {
         assertEquals(actual.getEventTtl(), expected.getEventTtl());
     }
 
-    @Test(expectedExceptions=IllegalArgumentException.class, expectedExceptionsMessageRegExp=".* EventTtl duration should be within 365 days .*")
-    public void testSubscriptionJsonWithEventTtlGreaterThan365Days() throws Exception {
-        Date now = new Date();
-        String nowString = DateTimeFormatter.ISO_INSTANT.withZone(ZoneOffset.UTC).format(now.toInstant());
-
-        String subscriptionJsonString = "{" +
-                "\"name\":\"test-subscription\"," +
-                "\"tableFilter\":\"intrinsic(\\\"~table\\\":\\\"review:testcustomer\\\")\"," +
-                "\"expiresAt\":\"" + nowString + "\"," +
-                "\"eventTtl\":31622400000" +
-                "}";
-
-        JsonHelper.fromJson(subscriptionJsonString, Subscription.class);
-    }
-
     @Test(expectedExceptions=IllegalArgumentException.class, expectedExceptionsMessageRegExp=".* EventTtl must be >0 .*")
     public void testSubscriptionJsonWithNegativeEventTtl() throws Exception {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.qa.bazaarvoice.com/coding_rules#rule_key=squid%3AS00112)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.lang.Exception', as it cannot be thrown from method's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.qa.bazaarvoice.com/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)